aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -393,22 +423,49 @@ The Hosting Bundle installer logs for the module are found at *C:\\Users\\%UserN
 
    * %windir%\SysWOW64\inetsrv\aspnetcore.dll
 
+::: moniker range=">= aspnetcore-2.2"
```

Comment:
Should the prior paths (existing in the topic above) be versioned for < 2.2? If not, no worries.